dbscripts/xml/upgrade/3.0.0_versioning.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 3.0.0_versioning.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 3.0.0.0 versioning updates.
+  -->
+<data>
+	<sql>
+		<!-- submission_settings -->
+		<query>UPDATE submission_settings SET submission_revision = 1 WHERE submission_revision IS NULL OR submission_revision = 0</query>
+		<!-- authors -->
+		<query>UPDATE authors SET version = 1 WHERE version IS NULL OR version = 0</query>
+		<query>UPDATE author_settings SET version = 1 WHERE version IS NULL OR version = 0</query>
+		<query>ALTER TABLE authors DROP PRIMARY KEY, ADD PRIMARY KEY (author_id, version)</query>
+		<query>UPDATE submission_galley_settings SET version = 1 WHERE version IS NULL OR version = 0</query>
+		<query>UPDATE submission_galley_files SET version = 1 WHERE version IS NULL OR version = 0</query>
+		<query>ALTER TABLE submission_galleys DROP COLUMN file_id</query>
```

Comment:
S. also comment above how to solve the issue when you need information before the DB XML schemas are executed, so that you would not need this code...